f8a_worker/utils.py

```diff
@@ -669,5 +670,20 @@ def get_session_retry(retries=3, backoff_factor=0.2, status_forcelist=(404, 500,
     session.mount('http://', adapter)
     return session
 
-# get not hidden files from current directory
-# print(list(get_all_files_from('.', file_filter=lambda a: not startswith(a, ['.']))))
+
+def normalize_package_name(ecosystem, name):
```

Comment:
Any particular reason why this is here? I think it should be placed on API server side instead as core should work directly on real package names.

Reply:
Normalizing package names on server side would work just fine for `golang`, but we need to be able to normalize package names in init task for other ecosystem. So I'd keep the logic at one place.